pkg/constants/constants.go

```diff
@@ -76,6 +76,11 @@ var (
 	PodMutatorWebhookName                  = KFServingName + "-pod-mutator-webhook"
 )
 
+// Request Logging Constants
+var (
+	LoggerDefaultUrl = getEnvOrDefault("REQUEST_LOGGING_DEFAULT_ENDPOINT", "http://default-broker")
```

Comment:
Can we document this environment variable?

Reply:
Have now added an explanation to the docs for the main request logging example

Reply:
@ryandawsonuk can we move this option to the configmap [logger section](https://github.com/kubeflow/kfserving/blob/master/config/configmap/inferenceservice.yaml#L98) ? I think environment variable is a bit hard to discover

Reply:
Makes sense, will have a look

Reply:
@yuzisun have now made this update